ynh-dev

```diff
@@ -108,18 +111,30 @@ elif [ "$1" = "run" ]; then
     echo ""
 
     # Get vagrant box info from version
-    if [ "$VERSION" = "stable" ]; then
-        BOX_NAME="yunohost/jessie-stable"
-        BOX_URL="https://build.yunohost.org/yunohost-jessie-stable.box"
+    if [ "$VERSION" = "stable64" ]; then
+        BOX_NAME="yunohost/jessie-amd64-stable"
+        BOX_URL="https://build.yunohost.org/yunohost-jessie-amd64-stable.box"
         IP="192.168.33.80"
-    elif [ "$VERSION" = "testing" ]; then
-        BOX_NAME="yunohost/jessie-testing"
-        BOX_URL="https://build.yunohost.org/yunohost-jessie-testing.box"
+    elif [ "$VERSION" = "testing64" ]; then
+        BOX_NAME="yunohost/jessie-amd64-testing"
+        BOX_URL="https://build.yunohost.org/yunohost-jessie-amd64-testing.box"
         IP="192.168.33.81"
-    elif [ "$VERSION" = "unstable" ]; then
-        BOX_NAME="yunohost/jessie-unstable"
-        BOX_URL="https://build.yunohost.org/yunohost-jessie-unstable.box"
+    elif [ "$VERSION" = "unstable64" ]; then
+        BOX_NAME="yunohost/jessie-amd64-unstable"
+        BOX_URL="https://build.yunohost.org/yunohost-jessie-amd64-unstable.box"
         IP="192.168.33.82"
+    elif [ "$VERSION" = "stable32" ]; then
+        BOX_NAME="yunohost/jessie-i386-stable"
+        BOX_URL="https://build.yunohost.org/yunohost-jessie-i386-stable.box"
```

Comment:
And I built them on my laptop also if needed, using the code in the PR you mentioned